db.json

```diff
@@ -4275,13 +4404,17 @@
   },
   "application/vnd.oasis.opendocument.formula-template": {
     "source": "iana",
-    "extensions": ["odft"]
+    "extensions": ["odft","otf"]
   },
```

Comment:
https://www.iana.org/assignments/media-types/application/vnd.oasis.opendocument.formula-template only lists .otf, not the pre-existing .otft